src/amo/components/SectionLinks/index.js

```diff
@@ -124,11 +135,15 @@ export class SectionLinksBase extends React.Component<Props> {
                 </Link>
               </DropdownMenuItem>
             )}
-            <DropdownMenuItem>
-              <Link className="SectionLinks-dropdownlink" to="/search-tools/">
-                {i18n.gettext('Search Tools')}
-              </Link>
-            </DropdownMenuItem>
+
+            {// See: https://github.com/mozilla/addons-frontend/issues/8680
+            _config.get('enableFeatureRemoveSearchTools') !== true && (
```

Comment:
I'm sure I'm missing something obvious here, but why not just `!_config.get('enableFeatureRemoveSearchTools')`?